### PR TITLE
Add automated_email and name to README

### PR DESCRIPTION
**Pull request by SuvigyaJain1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ So far, the following projects have been integrated to this repo:
 |--|--|
 |[File Encrypt Decrypt](https://github.com/hastagAB/Awesome-Python-Scripts/tree/master/file-encrypt-decrypt)|[Aditya Arakeri](https://github.com/adityaarakeri)|
 | [Address locator](https://github.com/hastagAB/Awesome-Python-Scripts/tree/master/Location_Of_Adress) | [Chris]() |
+| [Automated emails](https://github.com/hastagAB/Awesome-Python-Scripts/tree/master/automated_email) | [Suvigya]() |
```
Comment (hastagAB) [inline README.md]: Add Link to your GitHub profile.
`[Suvigya](Your PRofile Link)`

**Comment by SuvigyaJain1** [comment]
Is this right?

**Comment by hastagAB** [comment]
> Is this right?

Yes. 
Please squash your commits so that I can merge it.

**Comment by SuvigyaJain1** [comment]
yo its done! Thanks so much for your patience and help :) I think I've learnt the basics of git now.

**Comment by hastagAB** [comment]
> yo its done! Thanks so much for your patience and help :) I think I've learnt the basics of git now.

That's what Hacktoberfest is all about, to introduce people to the world of Open Source.
You have done great work. 
Keep contributing and best of Luck! :+1:

**Pull request merged**